backend/Entities/Services/DoctorService.cs

```diff
@@ -89,13 +87,12 @@ public List<string[]> GetDoctorsByProfessionId(int professionId)
             var str = _dbContext.ExecuteSqlQuery(cmd, '*', param);
             var values = str.Split('*');
             var list = new List<string[]>();
-            for (int i = 0; i < (values.Length - 1); i += 2)
+            for (int i = 0; i < (values.Length - 1); i += 3)
             {
-                string[] name = { values.GetValue(0 + i).ToString(), values.GetValue(1 + i).ToString() };
+                string[] name = { values.GetValue(0 + i).ToString(), values.GetValue(1 + i).ToString(),values.GetValue(2+i).ToString()};
```

Comment:
actually you may overhaul our data access logic. It would be much better if we pass objects from beginning, not the large concatenated string. it is the implication, not the problem itself